source/schemas/types.txt

```diff
@@ -509,3 +509,80 @@ with the ``title`` field set to the embedded object's type name:
             }
       }
    }
+
+.. _geospatial:
+
+Geospatial Data
+---------------
+
+Geospatial data describes points and other data on the earth's surface. App 
+Services supports defining GeoJSON Point data in a schema. 
+
+GeoJSON Point
+~~~~~~~~~~~~~
+
+A GeoJSON Point is a single location on the Earth's surface, described by its 
+longitude and latitude. To ensure wide compatibility, App Services stores these 
+points in accordance to the `GeoJSON spec <https://datatracker.ietf.org/doc/html/rfc7946>`_.
+To store a GeoPoint in Atlas, you must follow these rules:
+
+- It must be of type ``object``
+- It must be embedded within another type
+- It must contain the following fields:
+
+  - ``type``, which is always set to "Point"
+  - ``coordinates``, of type ``array``. The ``coordinates`` array contains 
+    exactly 2 double values. The first double is the longitude of the point and 
+    the second is the latitude.
+
+- Additional properties are allowed.
+ 
+The following is the schema of an object where the "location" property is 
+a GeoPoint:
+
+.. code-block:: json
+
+   {
+     "title": "MyObject",
+     "properties": {
+       "location": {
+         "bsonType": "object",
+         "required": [ "type" ],
+         "properties": {
+           "type": {
+             "bsonType": "string",
+             "const": "Point"
+           },
+           "coordinates": {
+             "bsonType": "array",
+             "items": {
+               "bsonType": "double",
+                "minItems": 2,
+                "maxItems": 2
+             }
+           }
+         }
+       },
+       "additional_location_property": {
+             "bsonType": "string"
+       },
+     }
+   }
+
+Querying GeoJSON Points
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can query geospatial points when using :ref:`Flexible Sync <flexible-sync>`. 
+No additional configuration is required. 
+
+To learn about querying against geospatial data, see:
```

Comment:
I think it could be cool to have a demo/walkthrough app where it tells you how to setup a Device Sync app from start to finish that is querying on geospatial data. Perhaps out of the scope of this ticket though

Reply:
Yes, cool and way beyond the scope. :) I have built one, so perhaps at some point I can water it down to use as a sample.